app/views/metadata_fields/_countryside_stewardship_grants.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="form-group">
+  <%= f.label :grant_type %>
+  <%= f.select :grant_type, f.object.facet_options(:grant_type), {}, { class: 'form-control' } %>
+  </div>
+<div class="form-group">
+  <%= f.label :land_use %>
+  <%= f.select :land_use, f.object.facet_options(:land_use), {}, { class: 'select2 form-control', multiple: true, data: {placeholder: 'Select a location'} } %>
+</div>
+<div class="form-group">
+  <%= f.label :tiers_or_standalone_items %>
+  <%= f.select :tiers_or_standalone_items, f.object.facet_options(:tiers_or_standalone_items), {}, { class: 'select2 form-control', multiple: true, data: {placeholder: 'Select a Tier or Standalone Item'} } %>
+</div>
+<div class="form-group">
+  <%= f.label :funding_amount %>
+  <%= f.select :funding_amount, f.object.facet_options(:funding_amount), { label: 'Funding (per unit per year)'}, { class: 'select2 form-control', multiple: true, data: {placeholder: 'Select a level of funding (per unit per year) '} } %>
```

Comment:
This line could probably do with breaking over several lines. I think it's pretty hard to follow as-is.